src/syntax/FStar.Syntax.Syntax.fsti

```diff
@@ -511,6 +511,7 @@ type sigelt' =
                           * lident                     //the inductive type of the value this constructs
                           * int                        //and the number of parameters of the inductive
                           * list lident               //mutually defined types
+			  * list attribute             // TODO document
```

Comment:
Why do we need this new field, why not use the `sigattrs` field in the type `sigelt`? (We create a sigelt for each data constructor as well.)

Reply:
Oh thanks for the comment, I actually completely missed that! (did the change in the surface AST then reflect that without enough thinking)
That simplifies things quite a lot!